src/provider/handler/getPosts.js

```diff
@@ -25,7 +41,7 @@ export function getPosts(params) {
       let mediaPromises = [];
       result.entry.forEach(result=>{
```

Comment:
please add [prettier](https://github.com/prettier/prettier) to the project for consistent code formatting, and enable auto format on save in your editor